README.md

```diff
@@ -147,68 +381,155 @@ For both these groups, containers present a unique challenge, as it seems almost
 |Continuous Deployment | Comes down to pushing containers to registries for production systems | No best practice established, but can interact with resources in some situations to deploy|
-|Monitoring| Monitoring is well integrated into services | Must "roll your own" monitoring, but traditionally DevOps services (e.g., Grafana, Prometheus) are used sometimes. |
+|Monitoring| Monitoring is well integrated into services | Must "roll your own" monitoring, but DevOps services (e.g., Grafana, Prometheus) are used sometimes. |
 | Security | DevSecOps is leading the way to make security an automated part of the development lifecycle | Security is unlikely to be automated, and a greater challenge with many users sharing the same space. |
 
+> we're very interested in addign things like scanning via gitlab. woudl argue that companies have
+> same issues if they aren't developing in the cloud -- intrnal devsecops aren't that great.
+
 ## HPC Technologies
 
 There are a family of technologies that are a core part of high performance computing that are not present, or are represented in a different way, on the cloud.
 **TODO** add more detail here.
 
+> we should talk about how this is a blurry line -- ParallelCluster is very similar to a regular
+> HPC environment.
+
 ## RSE-ops Best Practices
 
 Given the challenges and gaps above, we can now make an attempt to define RSE-ops best practices. This can mean anything from a software development approach, to an actual piece of software, to a cultural norm. In the case that a best practice does not exist, this presents an opportunity to work on it.
 
+> in many cases this is going to require buy-in from admins so the RSE side has to establish best
+> practices for both the deployment of the infra (gitlab, etc.) and the software (using the infra)
 
 ### Scientific Software
 
 Since HPC is optimized to build software natively, we require a package manager that can do so, and allow multiple different versions of software, or software requiring conflicting dependencies, to co-exist. [spack](https://github.com/spack/spack) is a package manager that was developed at Lawrence Livermore National Lab (LLNL) and is optimized for this use case.
 
+> PM's are emerging as kind of de-facto standards now which is nice, but few others are really
+> standards in the same collaborative way.
 
 ### Scaling
 
 **TODO** add MPI / scaling and other technologies here. Possibly flux?
 
+> IMO this is a huge category -- not sure we need to cover all of converged computing here but it's
+> related to workflow so probably valuable to talk about Flux, etc. -- Flux might be a good base
+> for RSE-ops tools.
+
 ### Software Distribution
 
  - [LMOD](https://lmod.readthedocs.io/en/latest/)
  - [spack](https://github.com/spack/spack)
- 
+
+> more at OS leve, also good ol' distributing source tarballs, and aren't HPC containers are in
```

Comment:
Maybe. I think I would consider a package manager to be something different than a container (which can contain a package manager). We will need to think about a good grouping.